components/multiselect/multiselect.jsx

```diff
@@ -235,42 +258,9 @@ export default class MultiSelect extends React.Component {
             }
         }
 
-        if (options && options.length > this.props.perPage) {
-            const pageStart = this.state.page * this.props.perPage;
-            const pageEnd = pageStart + this.props.perPage;
-            optionsToDisplay = options.slice(pageStart, pageEnd);
-            if (!this.props.loading) {
-                if (options.length > pageEnd) {
```

Comment:
We use the same `multiselect.jsx` component in few places so we will miss `next` and `previous` buttons in other places

<img width="534" alt="screenshot 2018-12-21 at 12 14 02 am" src="https://user-images.githubusercontent.com/4973621/50304221-687b1800-04b5-11e9-9800-5ff563451fac.png">

<img width="599" alt="screenshot 2018-12-21 at 12 15 23 am" src="https://user-images.githubusercontent.com/4973621/50304314-9fe9c480-04b5-11e9-9cf4-d702277d9521.png">

Can we make sure that we cover other places to have infinite loader as well?

Reply:
@sudheerDev Can you take a look a the questions above please?